<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Halturka</string>
  </property>
  <property name="windowIcon">
   <iconset resource="halturka.qrc">
    <normaloff>:/images/logotip.png</normaloff>:/images/logotip.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Work">
    <property name="title">
     <string>&amp;Work</string>
    </property>
    <addaction name="actionCustomers"/>
    <addaction name="actionWorkers"/>
    <addaction name="actionWorks"/>
    <addaction name="actionPayments"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="closeAction"/>
    <addaction name="closeAllAction"/>
    <addaction name="separator"/>
    <addaction name="tileAction"/>
    <addaction name="cascadeAction"/>
    <addaction name="separator"/>
    <addaction name="nextAction"/>
    <addaction name="previousAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Work"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhExclusiveInputMask</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCustomers"/>
   <addaction name="actionWorkers"/>
   <addaction name="actionWorks"/>
   <addaction name="actionPayments"/>
   <addaction name="actionStatistic"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="halturka.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
   <property name="softKeyRole">
    <enum>QAction::NoSoftKey</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionCustomers">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="halturka.qrc">
     <normaloff>:/images/customer.png</normaloff>:/images/customer.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;ustomers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionWorks">
   <property name="icon">
    <iconset resource="halturka.qrc">
     <normaloff>:/images/work.png</normaloff>:/images/work.png</iconset>
   </property>
   <property name="text">
    <string>Wo&amp;rks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionWorkers">
   <property name="icon">
    <iconset resource="halturka.qrc">
     <normaloff>:/images/worker.png</normaloff>:/images/worker.png</iconset>
   </property>
   <property name="text">
    <string>W&amp;orkers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPayments">
   <property name="icon">
    <iconset resource="halturka.qrc">
     <normaloff>:/images/payment.png</normaloff>:/images/payment.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;ayments</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="closeAllAction">
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="tileAction">
   <property name="text">
    <string>&amp;Tile</string>
   </property>
  </action>
  <action name="cascadeAction">
   <property name="text">
    <string>&amp;Cascade</string>
   </property>
  </action>
  <action name="nextAction">
   <property name="text">
    <string>Ne&amp;xt</string>
   </property>
  </action>
  <action name="previousAction">
   <property name="text">
    <string>Pre&amp;vious</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="halturka.qrc">
     <normaloff>:/images/preferences.png</normaloff>:/images/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStatistic">
   <property name="icon">
    <iconset resource="halturka.qrc">
     <normaloff>:/images/statistic.png</normaloff>:/images/statistic.png</iconset>
   </property>
   <property name="text">
    <string>Statistic</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="halturka.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
